### PR TITLE
Revert mapbox-gl to latest v1

### LINKED ISSUE #16185: Map interface/layout cannot be rendered after adding Mapbox Access Token in Project Settings
### Describe the Bug

Map interface does not work any more neither in interfaces or layouts.

After some research I was able to found that the issue is caused by the upgrade of `maplibre-gl` from `1.15.2` to `2.4.0` in #16052.

Here is more information about it:
https://github.com/maplibre/maplibre-gl-js/discussions/1226

### To Reproduce

- Provide a MapBox key in Project Settings
- Use map layout or create a map interface 

### Errors Shown

Within DevTools we can see the following on Console:
```
Fetch API cannot load mapbox://styles/directus/cl0bombrr001115taz5ilsynw. URL scheme "mapbox" is not supported.
```

### What version of Directus are you using?

9.19.2

### What version of Node.js are you using?

v16.14.2

### What database are you using?

postgis/postgis:13-3.1-alpine

### What browser are you using?

Chrome

### How are you deploying Directus?

running locally (main)

### PR DESCRIPTION
**Pull request by licitdev**
## Description

Fixes #16185. Revert update `mapbox-gl` from next major version.

## Type of Change

- [x] Bugfix
- [ ] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated. PR:

### CODE REVIEW

**Review by br41nslug — APPROVED** [review]
Can confirm this approach also loads the mapbox tiles correctly 😄

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -4021,6 +4021,14 @@ packages:
       xtend: 4.0.2
     dev: true
 
+  /@mapbox/mapbox-gl-supported/1.5.0_mapbox-gl@2.10.0:
```
Comment (joselcvarela) [inline pnpm-lock.yaml:4024]: It is working as expected now but why is `pnpm` doing this? 🤷

**Review by joselcvarela — APPROVED** [review]

File: pnpm-lock.yaml (hunk 1 above)
Comment (licitdev) [reply pnpm-lock.yaml]: Reverted the `mapbox-gl` required dependency to v1 which resolves the missing peer dependency.

Comment (joselcvarela) [reply pnpm-lock.yaml]: ✅ It works properly  on my end.
✅ I have also tested building from Docker and tested with reverse proxy (`localtunnel`) and it also works 👍

**Pull request merged**